src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/Extensions/ApplicationInsightsServiceCollectionExtensions.cs

```diff
@@ -41,7 +48,8 @@ public static IServiceCollection AddApplicationInsights(this IServiceCollection
             return services;
         }
 
-        public static IServiceCollection AddApplicationInsights(this IServiceCollection services)
```

Comment:
Are we removing an existing `public` method? Wouldn't that be a breaking change?

Reply:
Adding an overload - this method signature still exists line 34

Reply:
Yea, I missed that. Thanks!